src/xGadgetron/cGadgetron/gadgetron_data_containers.cpp

```diff
@@ -644,7 +644,7 @@ AcquisitionsFile::take_over(MRAcquisitionData& ac)
 }
 
 unsigned int 
-AcquisitionsFile::items()
+AcquisitionsFile::items() const
```

Comment:
cannot be const:

	unsigned int na = dataset_->getNumberOfAcquisitions();

and `getNumberOfAcquisitions` is not `const`.

Reply:
this works as we're accessing the method via a ptr. The ptr remains `const` (even if the object it points to isn't). A C++ quirck (but in this case a useful one).